### ISSUE TITLE
Performance on Chinese Text

### ISSUE THREAD

**Issue by sunlightsgy** [OptimalScale/LMFlow#5]
Has this framework been tested in Chinese? It is reported that LLaMA performed not well on corpus of Chinese.

**Comment by SHUMKASHUN** [comment]
We have not conducted extensive testing on Chinese, but we have tested some Chinese language questions and found that our trained model has some Chinese language conversation abilities. We put the results [online](http://3.84.211.127/) for your review. Please scroll down to the bottom of the page to see the Chinese examples.

There are good and bad examples. Here are some good examples. 
<img width="1191" alt="Screenshot 2023-03-29 at 3 27 08 AM" src="https://user-images.githubusercontent.com/31779666/228346230-b87ba55e-b50a-40d0-aa86-be1df5670bf1.png">
<img width="1305" alt="Screenshot 2023-03-29 at 3 27 52 AM" src="https://user-images.githubusercontent.com/31779666/228346381-5d09e73b-bb2e-4d9c-9cd6-b5d3390cf263.png">

We welcome you to use our code for larger-scale training and more testing, thank you.

**Comment by sunlightsgy** [comment]
The performance seems to be unsatisfactory. It is expected to see a foundation model pretrained (or finetuned) for Chinese specially. Of course I will try to finetune on my own dataset. Thanks for your efforts~

**Comment by shizhediao** [comment]
Hi,
Thanks for your feedback.
The initial goal of our open source toolkit is to provide a pipeline to help everyone finetune LLM rather than to get particularly good specific performance. It will be good if we can pre-train a large Chinese model from scratch, but the advantage of finetuning is faster training with fewer resources. 

I am very much looking forward to your trial of fine-tuning your own dataset and welcome to report new results. We also plan to do Chinese finetuning in the near future. The trained model and results will be open sourced soon.

Thanks
